packages/applications-service-api/src/controllers/subscriptions.js

```diff
@@ -46,14 +49,28 @@ const confirmSubscription = async (req, res) => {
 
 	validateSubscriptionDate(date);
 
-	await publishNSIPSubscription(caseReference, email, subscriptionTypes);
+	await publishCreateNSIPSubscription(caseReference, email, subscriptionTypes);
 
-	res.status(200).send();
+	res.send();
+};
+
+const deleteSubscription = async (req, res) => {
+	const { caseReference } = req.params;
+	const { email } = req.query;
+
+	const project = await getApplication(caseReference);
```

Comment:
Not a biggy but is it worth moving into it's own function?

Reply:
Thanks, have added a function and used it for both endpoints